libs/api/domains/fiskistofa/src/lib/models/quotaStatus.ts

```diff
@@ -0,0 +1,40 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
```

Comment:
```suggestion
@ObjectType('FiskistofaQuotaStatus`)
```

same for all of the other types.